packages/bits-ui/src/lib/bits/accordion/accordion.svelte.ts

```diff
@@ -337,8 +328,8 @@ class AccordionContentState {
  * CONTEXT METHODS
  */
 
-export const ACCORDION_ROOT_KEY = "Accordion.Root";
-export const ACCORDION_ITEM_KEY = "Accordion.Item";
+export const ACCORDION_ROOT_KEY = Symbol("Accordion.Root");
```

Comment:
I purposely put these back to strings from symbols because it kills our ability to give good error messages to the user regarding missing context parents. (see verifyContext)